### PR TITLE
Added bolus velocities to monthly output and eddy stats

### PR DESCRIPTION
**Pull request by lconlon**
Added monthly mean zonal and meridional bolus velocities as well as bolus velocity times temperature to eddy stats for Labrador Sea bias/ GM testing

### CODE REVIEW

File: your_changes.patch [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+             SSHSquared(iCell) = ssh(iCell)**2
+@@ -224,6 +230,8 @@ contains
+                velocityMeridionalSquared(k,iCell) = velocityMeridional(k,iCell)**2
+                velocityZonalTimesTemperature(k,iCell) = velocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
+                velocityMeridionalTimesTemperature(k,iCell) = velocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
++               velocityZonalTimesTemperature_GM(k,iCell) = GMBolusVelocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
++               velocityMeridionalTimesTemperature_GM(k,iCell) = GMBolusVelocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
+             end do
+          end do
+
```
Comment (vanroekel) [inline your_changes.patch]: @lconlon, please remove this from the commit history.

File: src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F [hunk 2]
```diff
@@ -224,7 +229,9 @@ subroutine ocn_compute_eddy_product_variables(domain, timeLevel, err)!{{{
                velocityMeridionalSquared(k,iCell) = velocityMeridional(k,iCell)**2
                velocityZonalTimesTemperature(k,iCell) = velocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
                velocityMeridionalTimesTemperature(k,iCell) = velocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
-            end do
+               velocityZonalTimesTemperature_GM(k,iCell) = GMBolusVelocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
```
Comment (vanroekel) [inline src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F]: I don't see where GMBolusVelocityZonal is being calculated.  I think this should be in mpas_ocn_time_integration_split, did you perhaps forget to commit changes to that file?  I would expect this code to give all zeros for this and the meridional array.

File: src/core_ocean/analysis_members/Registry_eddy_product_variables.xml [hunk 3]
```diff
@@ -39,6 +39,12 @@
 		<var name="velocityMeridionalTimesTemperature" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
 			description="cell-wise product of component of horizontal velocity in the northward direction and temperature"
 		/>
+                <var name="velocityZonalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
+                        description="cell-wise product of component of horizontal bolus velocity in the eastward direction and temperature"
+                />
+                <var name="velocityMeridionalTimesTemperature_GM" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1} C"
+                        description="cell-wise product of component of horizontal bolus velocity in the northward direction and temperature"
+                />
```
Comment (vanroekel) [inline src/core_ocean/analysis_members/Registry_eddy_product_variables.xml]: You also need changes to src/core_ocean/Registry.xml to add the GMbolusVelocityZonal and Meridional arrays.

File: src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F (hunk 2 above)
Comment (vanroekel) [reply src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F]: this is my mistake, ignore the comment.  The BolusZonal is already in ocn_diagnostics

File: src/core_ocean/analysis_members/Registry_eddy_product_variables.xml (hunk 3 above)
Comment (vanroekel) [reply src/core_ocean/analysis_members/Registry_eddy_product_variables.xml]: again, ignore here.

**Comment by lconlon** [comment]
bolus_velocities

File: your_changes.patch (hunk 1 above)
Comment (lconlon) [reply your_changes.patch]: Should be gone now. Thanks for catching that!

**Pull request closed without merging**